tests/snapshots/tests.contrib.django.test_django_snapshots.test_asgi_200_31.snap

```diff
@@ -0,0 +1,230 @@
+[[{"name" "django.middleware"
```

Comment:
this is missing the `django.request` span?

Reply:
Yep - that's the repro of https://github.com/DataDog/dd-trace-py/issues/2559

Reply:
oh, got it, yeah idk why I was expecting this to be fixed in this PR. this just adds the tests 🤦🏻